src/libratbag.c

```diff
@@ -1043,14 +1043,7 @@ LIBRATBAG_EXPORT bool
 ratbag_resolution_has_capability(struct ratbag_resolution *resolution,
 				 enum ratbag_resolution_capability cap)
 {
-	switch (cap) {
-	case RATBAG_RESOLUTION_CAP_SEPARATE_XY_RESOLUTION:
-		break;
-	default:
-		return 0;
-	}
-
-	return !!(resolution->capabilities & (1 << cap));
+	return !!(resolution->capabilities & (1 << cap)); // is this right?
```

Comment:
I guess the switch was just here because there was only one capability?

Reply:
I am not not sure why the case was there. @whot?

Reply:
I now know why, was a replacement for assert.